### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by eacharles**
Added a few sentence to contributing.rst to better capture our expectations.

### CODE REVIEW

**Review by sschmidt23 — APPROVED** [review]
Looks good, thanks for fixing this up

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -4,13 +4,28 @@ Contributing
...
+the issue's discussion page to let others know you're working on it. 
+Then, make a branch with a name of the
+form `issue/[#]/brief-description` and make changes in your branch. 
+While developing in a branch, don't forget to pull from `main` regularly to make sure your work is compatible with other recent changes.
+
+Before you make a pull request we ask that you do two things:
+   1. Run `pylint` and clean up the code accordingly.  You may need to
+      install `pylint` to do this.
+   2. Add unit tests and make sure that the new code is fully
+      `covered`.   You make need to install `pytest` and `pytest-cov`
```
Comment (sjoudaki) [inline docs/source/contributing.rst]: Very minor: "make need" --> "may need"?

**Comment by sjoudaki** [comment]
@eacharles So sorry for the delay, aside from the minor comment above, this looks great!

**Review by sjoudaki — APPROVED** [review]

File: docs/source/contributing.rst (hunk 1 above)
Comment (sjoudaki) [inline docs/source/contributing.rst]: Sorry, just one more comment: we might want to detail/specify what is an acceptable unit test coverage (in line with the rest of RAIL)? In other words, we might want to specify what percentage we consider to be "fully" covered?

Comment (eacharles) [reply docs/source/contributing.rst]: Yep, I added some text about that.

Comment (sjoudaki) [reply docs/source/contributing.rst]: Thanks!

**Comment by sjoudaki** [comment]
@aimalz @eacharles It seems we still have a typo of the form (commented above): "You make need to install" which should say "You may need to install".

**Pull request merged**